portal-api/src/v1/validation/validateIds.js

```diff
@@ -60,6 +60,22 @@ const isValidMonth = (month) => Number.isInteger(month) && month >= 1 && month <
  */
 const isValidYear = (year) => Number.isInteger(year) && year >= 2000 && year <= 2100;
 
+/**
+ * isValidReportPeriod
+ * checks if the supplied report period is a valid report period object
+ * @param {unknown} reportPeriod
+ * @returns {boolean}
+ */
+const isValidReportPeriod = (reportPeriod) => !!(
```

Comment:
!! (Boolean conversation) is deprecated, compound `&&` without brackets will suffice.

Reply:
If we want to ensure a `boolean` return type then we either need the `!!` type cast or we will need to change to explicit `=== undefined` checks to make sure the return type is `boolean`. If you just use compound `&&` in this case, the JSDocs return type would need to be updated to say that the function can return `null | undefined | boolean`. Which approach would you prefer?

Reply:
Can we use `Boolean()`?